docs/source/index.rst

```diff
@@ -4,7 +4,7 @@ What is Pitt Broker
 ===================
 
 **Pitt Broker** is a cloud-based, alert distribution service designed to provide
```

Comment:
I suggest we change "Pitt Broker" to "Pitt-Google Broker" (4 instances in this file).

Reply:
Agreed